apps/i18n/common/en_us.json

```diff
@@ -1001,6 +1001,8 @@
   "emptyTextResponsesTable": "Most of our classes are designed to work for younger ages and do not require students to type text responses to questions, so you will not see any text responses here. For older students in middle and high school courses, you can see their submissions to open-ended text response questions by choosing the assigned course from the drop down above.",
   "emptyTopLevelBlock": "There are no blocks to run. You must attach a block to the {topLevelBlockName} block.",
   "enable": "Enable",
+  "enableAITutor": "Enable AI Tutor",
+  "enableAITutorTooltip": "Turning this on will give students in your section access to AI Tutor on certain AP CSA levels.",
```

Comment:
@moneppo I wasn't too precious about these exact strings, but if you have preferences for what exactly they should say happy to incorporate now or revise later.